tracing-futures/Cargo.toml

```diff
@@ -18,17 +18,19 @@ keywords = ["logging", "profiling", "tracing", "futures", "async"]
 license = "MIT"
 
 [features]
-default = ["std-future"]
+default = ["std-future", "std"]
 futures-01 = ["futures_01"]
 futures-03 = ["std-future", "futures", "futures-task"]
 std-future = ["pin-project"]
+std = ["tracing/std"]
+unsafe_global = ["tracing/unsafe_global"]
```

Comment:
I don't entirely understand why `tracing-futures` needs to expose its own feature flag for this?

Reply:
yeah, this should go in #497 and I should have splitted both more clearly.

Reply:
better this way.